builder-support/debian/recursor/debian-buster/rules

```diff
@@ -41,15 +41,20 @@ override_dh_auto_install:
 	install -m 644 -t debian/pdns-recursor/usr/share/pdns-recursor/snmp RECURSOR-MIB.txt
 	rm -f debian/pdns-recursor/etc/powerdns/recursor.conf-dist
 	rm -f debian/pdns-recursor/etc/powerdns/recursor.yml-dist
-	./pdns_recursor --no-config --config=default | sed \
-		-e 's!^# config-dir=.*!config-dir=/etc/powerdns!' \
-		-e 's!^# hint-file=.*!&\nhint-file=/usr/share/dns/root.hints!' \
-		-e 's!^# include-dir=.*!&\ninclude-dir=/etc/powerdns/recursor.d!' \
-		-e 's!^# local-address=.*!local-address=127.0.0.1!' \
-		-e 's!^# lua-config-file=.*!lua-config-file=/etc/powerdns/recursor.lua!' \
-		-e 's!^# quiet=.*!quiet=yes!' \
-		-e '/^# version-string=.*/d' \
-		> debian/pdns-recursor/etc/powerdns/recursor.conf
+	@echo "\
+	dnssec:\n\
+	  # validation: process\n\
+	recursor:\n\
+	  hint_file: /usr/share/dns/root.hints\n\
+	  include_dir: /etc/powerdns/recursor.d\n\
+	  lua_config_file: /etc/powerdns/recursor.lua\n\
+	incoming:\n\
+	 # listen:\n\
+	 # - 127.0.0.1\n\
```

Comment:
Shouldn't these be uncommented to match the previous configuration?

Reply:
Also unless I'm missing something, which is quite possible, we are no longer setting some options?

Reply:
I decided comment the line settings equal to the default settings, but leave (some) in to show how you set some common settings, like the listen address.